### PR TITLE
Create introduction/conclusion.md

### PR DESCRIPTION
**Pull request by rrrutledge**
This file holds the text to accompany the [Conclusion](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321611) video.
This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).

### CODE REVIEW

File: introduction/conclusion.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Inner source applies open source best practices and principles to internal software development.
+It gives an additional option to consumers when producing teams are not able to deliver a needed feature request.
+Successful inner source involves a **product owner** and **trusted committer** from the **host team** as well as a **contributor** from the **guest team**.
+Done effectively, inner source brings many benefits to both participating teams.
+They key principles upon-which effective inner source works are **voluntary code contribution** and **prioritized mentorship**.
+
+While this training contains a high-level overview of inner source, there are many more details useful in making inner source actually work for your team.
+If you want to stay connected to the ongoing conversation around inner source and its best practices, then join [the InnerSource Commons](http://innersourcecommons.org).
+The commons sponsors a Slack channel, an inner source patterns working group, and multiple in-person summits each year.
+We're also planning to introduce more segments in this learning path outlining detail around the product owner, trusted committer, and contributor roles.
```
Comment (gruetter) [inline introduction/conclusion.md]: We should reference both the PO and TC videos here.

Comment (rrrutledge) [reply introduction/conclusion.md]: Removed this text as it no longer feels as relevant as it did when written a year ago 😄.

Comment (rrrutledge) [reply introduction/conclusion.md]: Also  see https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/6#discussion_r248257523

**Comment by rrrutledge** [comment]
Things look good? Click the ✅❓

**Comment by lenucksi** [comment]
Pinging #41 as this article will definitely need some hyperlinks. Other than that: Meeerge!

**Pull request merged**